src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/FilterClauseDeserializer.java

```diff
@@ -299,7 +299,14 @@ private List<ComparisonExpression> createComparisonExpressionList(
           || operator == ValueComparisonOperator.GTE
           || operator == ValueComparisonOperator.LT
           || operator == ValueComparisonOperator.LTE) {
+        // Note, added 'valueObject instanceof String || valueObject instanceof Boolean', this is to
+        // unblock some table filter against non-numeric column
+        // e.g. {"event_date": {"$gt": "2024-09-24"}}, {"is_active": {"$gt": true}},
+        // {"name":{"$gt":"Tim"}}
+
         if (!(valueObject instanceof Date
+            || valueObject instanceof String
```

Comment:
@tatu-at-datastax  @amorton 
This is the quick special handling we talked about last week. 
To unblock table filter against non-numeric column.